Src/Sarif.PatternMatcher/AnalyzeContext.cs

```diff
@@ -13,6 +13,16 @@ public class AnalyzeContext : AnalyzeContextBase
     {
         public static long FilesFilteredBySniffRegex;
 
+        /// <summary>
+        /// Gets or sets a dictionary linking file text with
+        /// A String8 that is used to in RE2 searching.
```

Comment:
this isn't grammatically correct.

Reply:
This falls under the should the 'if you touch it, you own it' rule.

Ultimately if we decide we don't care about the order then this would not have come up and there are literally hundreds of comments I'm guessing that need help and that takes away time from perhaps other higher yield tasks or  perhaps a dedicated cleanup pass for comments.

Reply:
Yes, you touched it. When you touch code, in our project, people should take a careful look at it. 

Here is an updated code comment. I invested more time in discussing this with you than authoring it. And btw - improving code comments is a high yield task in my opinion. :)


    /// <summary>
   /// Gets or sets a map that associates file text with data used in RE2
   /// searching, specifically, a String8 form of the file text and arrays
   /// that comprise a map of UTF8 to UTF16 byte indices. This data is
   /// required to rationalize RE2 match segments with original .NET strings.